CONTRIBUTING.md

```diff
@@ -1,23 +1,25 @@
-# Web Platform Incubator Community Group
+# Contributing
 
-This repository is being used for work in the W3C Web Platform Incubator Community Group, governed by the [W3C Community License
-Agreement (CLA)](http://www.w3.org/community/about/agreements/cla/). To make substantive contributions,
-you must join the CG.
+Contributions to this repository are intended to become part of Recommendation-track documents
+governed by the [W3C Patent Policy](http://www.w3.org/Consortium/Patent-Policy-20040205/) and
```

Comment:
> Hm, allowing warnings makes me sad. My vote would be to add an exception in bikeshed for "notes". My second choice would be to add boilerplate privacy/security sections that say something like "Please see (link to main spec section)".

First option probably goes a bit too far. These particular notes do not need a dedicated section because the main spec will most likely contain all relevant considerations, but I think that people in privacy and security groups would argue that notes need such a section otherwise.

So second option could be good. I haven't tried it, but if text is going to be the same for all registry files, we can probably achieve that with a [file-based include](https://tabatkins.github.io/bikeshed/#bp-include) as well. Want me to give it a try?

> 
> Are we not able to set status to WG-NOTE as part of the transition? Alternatively, we could just continue to use NOTE? Not sure what effect this has on the generated "intended to become" text.

The rule of thumb is that documents in a GitHub repository are all "Editor's Drafts" (or "Community Group Drafts" for Community Group deliverables). The other statuses such as "Working Group Note", "Recommendation", "Working Draft", etc. are for the versions of these documents published under "/TR/".

Having a `WG-NOTE` status would make Bikeshed use the `status-WG-NOTE.include` (which is also part of the PR I prepared).